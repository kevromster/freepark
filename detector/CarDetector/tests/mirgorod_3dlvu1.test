config = camera_nsk_mirgorod_3dlvu.txt
input = input/mirgorod_3dlvu1.png
expected = expected/mirgorod_3dlvu1.out
