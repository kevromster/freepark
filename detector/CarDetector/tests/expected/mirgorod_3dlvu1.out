[
parking id: 24
spots count: 1

{ lat: 54.853769; lon: 83.040094; complexity: NEWBIE }
]

[
parking id: 25
spots count: 1

{ lat: 54.85366; lon: 83.040178; complexity: ACCURATE }
]

[
parking id: 26
spots count: 3

{ lat: 54.853745; lon: 83.040042; complexity: NEWBIE }
{ lat: 54.853702; lon: 83.040081; complexity: NEWBIE }
{ lat: 54.85366; lon: 83.04012; complexity: NEWBIE }
]

